tests/unit/NetworkConnection.test.js

```diff
@@ -12,6 +12,7 @@ suite('NetworkConnection', function() {
     this.completeSync = sinon.stub();
     this.removeEntitiesOfClient = sinon.stub();
     this.updateEntity = sinon.stub();
+    this.updateEntityMulti = sinon.stub();
```

Comment:
Were you actually able to run the tests? That would be pretty great if they passed.

Reply:
kinda sorta. test suite passes on windows for me, fails on linux, and fails in travis for different reasons :) I think I can get it working though.